recipe/meta.yaml

```diff
@@ -54,10 +54,10 @@ about:
     Analysis of dose-response data is made available through a suite of flexible and versatile
     model fitting and after-fitting functions.
   license_family: GPL2
-  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-2'  # [unix]
-  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-2'  # [win]
```

Comment:
Can you try just `LICENCE`.

Reply:
Sure

Reply:
I wonder if the cran skeleton should be tweaked to make that automatic.

Reply:
To my knowledge, this is not standard and the pkg is "broken".